snowflake/Makefile

```diff
@@ -7,9 +7,22 @@ all: bin
 bin:
 	$(GO) build -o $(BINDIR)/theia-sf antrea.io/theia/snowflake
 
-.PHONY: test
-test:
-	$(GO) test -v ./...
+.PHONY: .coverage
+.coverage:
+	mkdir -p $(CURDIR)/.coverage
+
+.PHONY: test-unit
+test-unit: .coverage
+	@echo
+	@echo "==> Running unit tests <=="
+	$(GO) test -race -coverpkg=antrea.io/theia/snowflake/cmd/...,antrea.io/theia/snowflake/pkg/... \
+	  -coverprofile=.coverage/coverage-unit.txt -covermode=atomic \
+	  antrea.io/theia/snowflake/cmd/... antrea.io/theia/snowflake/pkg/... 
+
+.PHONY: generate
+generate:
+	MOCKGEN_COPYRIGHT_FILE=$(PWD)/hack/boilerplate/license_header.raw.txt \
```

Comment:
please use `$(CURDIR)` like we do in other places in this Makefile

https://stackoverflow.com/questions/52437728/bash-what-is-the-difference-between-pwd-and-curdir

Reply:
thanks, addressed